vaadin-core/pom.xml

```diff
@@ -34,6 +34,12 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>vaadin-prod-bundle</artifactId>
```

Comment:
Prod bundle is always added, but would be nice to drop it in dev mode and when a custom bundle is generated.

Reply:
Can it be a dependency of the maven and gradle plugins instead of Flow?

Reply:
I assumed that for using the default prod bundle, no one need to add `vaadin-maven-plugin:build-frontend`, because no frontend build is needed. Thus, if I include it to the plugin module...it wouldn't be included into class-path.

Reply:
You run the production build in the same way also in the future

Reply:
So you think that `build-frontend` goal is still needed for running the app with default prod bundle?

Reply:
Yes. Even if it is a no-op in some case. That is the place where you would decide what you need to do for the production package

Reply:
How can I add it exclusively for `build-frontend` goal? `vaadin-maven-plugin` is used in dev mode with `prepare-frontend`, so it will have the default bundle anyway.

Reply:
Why would it have to be exclusively for that? If it is a plugin dependency, you need to copy the files during the production build anyway

Reply:
Done, now it's a dependency of plugins